docs/deployment/version_skew_policy.md

```diff
@@ -0,0 +1,70 @@
+# Version Skew Policy
+
+This document describes the maximum version skew supported between various Gardener components.
+
+## Supported Gardener Versions
+
+Gardener versions are expressed as `x.y.z`, where `x` is the major version, `y` is the minor version, and `z` is the patch version, following Semantic Versioning terminology.
+
+The Gardener project maintains release branches for the most recent three minor releases.
+
+Applicable fixes, including security fixes, may be backported to those three release branches, depending on severity and feasibility.
+Patch releases are cut from those branches at a regular cadence, plus additional urgent releases, when required.
+
+For more information, see [this document](../development/process.md#releases).
+
+### Supported Version Skew
+
+Technically, we follow the same [policy](https://kubernetes.io/releases/version-skew-policy/) as the Kubernetes project.
+However, given that our release cadence is much more frequent compared to Kubernetes (every `14d` vs. every `120d`), in many cases it is possible to skip a version.
+Still, to be on the safe side, it is highly recommended following the described policy.
```

Comment:
```suggestion
However, given that our release cadence is much more frequent compared to Kubernetes (every `14d` vs. every `120d`), in many cases, it is possible to skip a version.
Still, to be on the safe side, it is highly recommended to follow the described policy.
```

Reply:
While I switched `following` to `to follow`, the `,` seems wrong, so I haven't added it.